test/functionalTest/cases/0945_pagination_with_cprs/pagination_with_cprs_mix_entities2.test

```diff
@@ -0,0 +1,861 @@
+# Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+# This is a variant of pagination_with_cprs_mix_entities.test but with the
+# en4 entity (the one with an attribute registered for the CPr) in the middle
+# by creDate, instead that at the end
```

Comment:
This comment block is probably more visible if in the --SHELL-- section, right before the  01, 02 etc.